docs/docs/user-guide/using-cli-context.md

```diff
@@ -0,0 +1,107 @@
+# Use CLI for Microsoft 365 context
+
+CLI for Microsoft 365 context provides unique functionality to save any kind of option and its value you might use for any command. The options saved in the context may be used in any kind of command execution, that way you may provide less or no options. It may be especially useful in scenarios where you many times use the same option in multiple commands or you would like to group all commands parameters in a single file that are used in the a script file.
+
+## How to get started
+
+In order to create an empty context we may execute the following command:
+```powershell
+m365 context init
+```
+
+To add or update an option in the context use the `m365 context option set` command. The `name` is used to define the option name and `value` is used to define its default value. For example, if we want to set `test list` value for option `listTitle` we should execute:
+```powershell
+m365 context option set --name 'listTitle' --value 'test list'
+```
+
+In order to check what is defined in the context we may use the `m365 context option list` command.
+
+To remove a specific option defined in the context we may run:
+```powershell
+m365 context option remove
+```
+
+In order to remove the full context we may execute the following command:
+```powershell
+m365 context remove
+```
+
+## How does it work
+
+When a command is executed CLI will first check for the `.m365rc.json` file in the current directory (this is where the context is saved once you set any option). If present, CLI will parse the context object and check if any of the options defined in it may be used for the currently executed command. If yes CLI will execute this command with the option and its value taken from the context.
+
```

Comment:
Is it an idea to include headers here?

```suggestion

### Example 1:
```

Reply:
actually, I don't agree with this one. Here the example is only added to provide more context to the above explanation 😉. The real full example is described below

Reply:
It's currently a bit odd to run into an examples header after having read an example 😀 
Do you have other ideas? Maybe include all the examples under the single Examples header?

Reply:
Or Let's just remove this example that provided more context 🤔?
O let's name the it?
`## Scenario`
🤔

Reply:
Yes, maybe removing it would be the best. It's already covered in the other example. 
Removing it makes the page more concise.